pandas/tests/series/test_timeseries.py

```diff
@@ -107,7 +107,7 @@ def test_shift(self):
         # incompat tz
         s2 = Series(date_range('2000-01-01 09:00:00', periods=5,
                                tz='CET'), name='foo')
-        pytest.raises(ValueError, lambda: s - s2)
+        pytest.raises(TypeError, lambda: s - s2)
```

Comment:
this is an API change

Reply:
as in needs to be noted in whatsnew or needs to be avoided?